crawl-ref/source/religion.cc

```diff
@@ -3938,6 +3953,13 @@ bool god_hates_spell(spell_type spell, god_type god, bool fake_spell)
     return false;
 }
 
+/**
+ * Will your god excommunicate you if you actually cast spell?
```

Comment:
Thank goodness for this docstring. Reading just the diff my first thought was, "Really, we have a distinction between loathes and hates? I wonder what the difference is."